test/acceptance/pageObjects/listUsersPage.js

```diff
@@ -0,0 +1,54 @@
+const util = require('util');
+
+module.exports = {
+	url: function () {
+		return this.api.launchUrl + 'user/list.php?leftmenu=users';
+	},
+
+	commands: [
+		{
+			listOfUsersDisplayed: async function (dataTable) {
+				const usersList = dataTable.hashes();
+
+				for (const row of usersList) {
+					let login = row['login'];
+					let lastName = row['last name'];
+					const userDetail = util.format(this.elements.userList.selector, login, lastName);
+
+					await this.useXpath()
+						.waitForElementVisible('@userRow')
+						.waitForElementVisible(userDetail);
+				}
+
+				return this.useCss();
+			},
```

Comment:
```suggestion
                              this.useXpath()
				for (const row of usersList) {
					let login = row['login'];
					let lastName = row['last name'];
					const userDetail = util.format(this.elements.userList.selector, login, lastName);

					await this.waitForElementVisible('@userRow')
						.waitForElementVisible(userDetail);
				}

				return this.useCss();
			},
```
HBT? :thinking:
You do not need the step useXpath() inside loop